NK: This message message means supervisord reaped a process that it didn't know about.
This is not an error, but it's not useful to log it as info in production.

--- supervisord.py
+++ supervisord.py.patched
@@ -276,7 +276,7 @@
             process = self.options.pidhistory.get(pid, None)
             if process is None:
                 _, msg = decode_wait_status(sts)
-                self.options.logger.info('reaped unknown pid %s (%s)' % (pid, msg))
+                self.options.logger.debug('reaped unknown pid %s (%s)' % (pid, msg))
             else:
                 process.finish(pid, sts)
                 del self.options.pidhistory[pid]